src/components/Tasklist/ToDoCard.js

```diff
@@ -30,42 +31,44 @@ export default function ToDoCard({
   }
 
   return (
-    <MainWrapper>
-      <TodoMain>
-        <ButtonGrab>
-          <img src={lines} alt="menu" />
-        </ButtonGrab>
-
-        {updateTask ? (
-          <Section>
-            <label htmlFor="edit" />
-            <EditInputField
-              onChange={handleChange}
-              type="text"
-              name="edit"
-              value={value}
-            ></EditInputField>
-            <EditCheckedButton onClick={handleClick}>
-              <img src={done} alt="menu" width="25" />
-            </EditCheckedButton>
-          </Section>
-        ) : (
-          <TaskList strikeThrough={completed}>{todo}</TaskList>
-        )}
-
-        <Checkbox
-          type="checkbox"
-          checked={completed}
-          onChange={() => onHandleIsChecked(id)}
-        />
-        <ButtonEdit onClick={updateTaskHandler}>
-          <img src={edit} alt="menu" width="30" />
-        </ButtonEdit>
-        <ButtonTrash onClick={() => onHandleDeleteTask(id)}>
-          <img src={trashIcon} alt="menu" width="25" />
-        </ButtonTrash>
-      </TodoMain>
-    </MainWrapper>
+    <SortableItem>
+      <MainWrapper>
+        <TodoMain>
+          <SortableKnob>
+            <ButtonGrab draggable="true">
+              <img src={lines} alt="menu" />
+            </ButtonGrab>
+          </SortableKnob>
+          {updateTask ? (
+            <Section>
+              <label htmlFor="edit" />
+              <EditInputField
+                onChange={handleChange}
+                type="text"
+                name="edit"
+                value={value}
+              ></EditInputField>
```

Comment:
You can write it like this:
```
<EditInputField
                onChange={handleChange}
                type="text"
                name="edit"
                value={value}
              />
```

Reply:
You are absolutely right, this makes the code more readable. I have adjusted it.